examples/client/tui/commands.nim

```diff
@@ -722,6 +722,72 @@ proc command*(self: Tui, command: SendMessage) {.async, gcsafe, nimcall.} =
   else:
     asyncSpawn self.client.sendMessage(command.message)
 
+# Call ----------------------------------------------------------------
+
+proc help*(T: type CallRPC): HelpText =
+  let command = "call"
+  HelpText(command: command, aliases: aliased[command], parameters: @[
+    CommandParameter(name: "input", description: "Options JSON")
+    ], description: "Calls an Ethereum RPC method")
+
+proc new*(T: type CallRPC, args: varargs[string]): T =
+  var
```

Comment:
Not sure if this is better, but it's possible to write `let input = if args.len > 0: args[0] else: ""`